config/redirects.txt

```diff
@@ -52,7 +52,6 @@
 ^\/using\/cli\/?(\?.+)?(#.+)?$ -> /reference/balena-cli/$1$2
 ^\/learn\/deploy\/release-strategy\/update-locking\/API\/?(\?.+)?(#.+)?$ -> /reference/supervisor/supervisor-api/$1$2
 ^\/reference\/cli\/?(\?.+)?(#.+)?$ -> /reference/balena-cli/$1$2
-^\/learn\/develop\/integrations\/google\/?(\?.+)?(#.+)?$ -> /learn/develop/integrations/google-iot/$1$2
```

Comment:
@kb2ma The previous integrations were quite old, but I think it would make sense for them to be redirected to the new page? Can you add 4 redirect rules to redirect the 4 integrations over to the new page.

Reply:
Fixed in 83b1b5b